### PR TITLE
Desc

### PR DESCRIPTION
**Pull request by bsonge**
#11 This takes an object of short, long, and header, all of which are optional, though you must have at least one of them.

### CODE REVIEW

File: app/components/Description/index.js [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+* Pass in text object with short and long descriptions as values {short:'', long:''}
+*/
+
+import React from 'react';
+// import styled from 'styled-components';
+
+import PropType from 'prop-types';
+import { Panel, Button } from 'react-bootstrap';
+// import { FormattedMessage } from 'react-intl';
+// import messages from './messages';
```
Comment (Quinn-Donnelly) [inline app/components/Description/index.js]: Use the Formatted Messages.

File: app/components/Description/index.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+class Description extends React.PureComponent { // eslint-disable-line react/prefer-stateless-function
+  constructor(props) {
+    super(props);
+    this.state = { expanded: false };
+    this.flip = this.flip.bind(this);
+  }
+  flip(evt) {
+    evt.preventDefault();
+    // this.expanded = !this.expanded;
```
Comment (Quinn-Donnelly) [inline app/components/Description/index.js]: If not needed remove

File: app/components/Description/index.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+    const button = <Button bsStyle="link" onClick={(evt) => { this.flip(evt); }}>{this.state.expanded ? 'Less' : 'More' }</Button>;
+    const body = () => {
+      if (short && long) { // if both are defined
+        return (<Panel.Body>{this.state.expanded ? long : short }{button}</Panel.Body>);
+      } else if (!short) {
+        return (<Panel.Body>{ long }</Panel.Body>);
+      }
+      return (<Panel.Body>{ short }</Panel.Body>);
+    };
+    // const toggleable = short && long;
```
Comment (Quinn-Donnelly) [inline app/components/Description/index.js]: If not needed remove.

File: app/containers/HomePage/index.js [hunk 4]
```diff
@@ -11,14 +11,24 @@
...
+
 import messages from './messages';
 
+
 export default class HomePage extends React.PureComponent { // eslint-disable-line react/prefer-stateless-function
   render() {
+    this.data = [{ str: 'hi', num: 1234, bool: 'true', wild: 'bye' },
+                { str: 'aahsadpfajwepoifjawoipjfawjef', num: null, bool: 'false', wild: 420 },
+                { str: 'aah', num: null, bool: null, wild: null }];
+    this.thing = 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer nec odio. Praesent libero. Sed cursus ante dapibus diam. Sed nisi. Nulla quis sem at nibh elementum imperdiet. Duis sagittis ipsum. Praesent mauris. Fusce nec tellus sed augue semper porta. Mauris massa. Vestibulum lacinia arcu eget nulla. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos. /n/nCurabitur sodales ligula in libero. Sed dignissim lacinia nunc. Curabitur tortor. Pellentesque nibh. Aenean quam. In scelerisque sem at dolor. Maecenas mattis. Sed convallis tristique sem. Proin ut ligula vel nunc egestas porttitor. Morbi lectus risus, iaculis vel, suscipit quis, luctus non, massa. Fusce ac turpis quis ligula lacinia aliquet. Mauris ipsum.';
```
Comment (Quinn-Donnelly) [inline app/containers/HomePage/index.js]: Revert test page to previous state.

**Review by Quinn-Donnelly — CHANGES_REQUESTED** [review]
I have detailed changes needed in my review.

**Review by Quinn-Donnelly — APPROVED** [review]
Looks good. :+1:

**Review by Jubstron — APPROVED** [review]
oh oH OH I'M ... I'M GOING TO - I'M GOING TO FUCKING APPROVEEEEEEEE :sweat_drops:

**Review by Jubstron — COMMENTED** [review]
:thinking: :thinking: :thinking: 
Where did the Description go?
:thinking: :thinking: :thinking:

**Review by wrejcodes — APPROVED** [review]
Looks good 👍

**Pull request merged**